.github/workflows/regression-tests.yml

```diff
@@ -168,6 +168,15 @@ jobs:
             cc: clang
             extra_name: "clang | "
 
+          - sim: icarus             # with gcov/gcovr
+            sim-version: v10_3
+            lang: verilog
+            python-version: 3.8
+            os: ubuntu-latest
+            extra_name: "gcovr | "
+            cxx: "ccache g++"
+            cc: "ccache gcc"
```

Comment:
Try this.
```suggestion
            cxx: "\"ccache g++\""
            cc: "\"ccache gcc\""
```

Reply:
Fails earlier. I guess I'll have to move this into ``tox.ini``.

Reply:
Apparently this isn't the correct way to escape quotes in YAML string values.